assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js

```diff
@@ -132,10 +134,11 @@ function DashboardAllTrafficWidget( { Widget, WidgetReportZero } ) {
 		reportArgs = {
 			'explorer-table.plotKeys': '[]',
 			'_r.drilldown': `analytics.pagePath:${ getURLPath( entityURL ) }`,
+			...generateDateRangeArgs( { startDate, endDate, compareStartDate, compareEndDate } ),
```

Comment:
You currently add the dates only in this `entityURL` clause, which only applies on the single URL dashboard view, where data is displayed not for the entire site but only a single URL. The date arguments need to be added unconditionally though, so you could put them into the `reportArgs` assignment above, and then in this clause, merge the `entityURL`-specific arguments into it.